examples/chat.rs

```diff
@@ -78,13 +78,30 @@ fn main() {
 
     // We create a custom network behaviour that combines floodsub and mDNS.
     // In the future, we want to improve libp2p to make this easier to do.
+    // Use the derive to generate delegating NetworkBehaviour impl and requires the
```

Comment:
```suggestion
    // Use the derive to generate delegating NetworkBehaviour impl and require the
```

Not a native speaker, but this sounds better to me.

Reply:
Thanks and not a native speaker either but the `s` definetly seems like a typo now that you pointed it out.